README.md

```diff
@@ -1 +1,156 @@
-# country-global
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+## 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+   - [Usage](#usage)
+- [👥 Authors](#authors)
+ - [🚀 Live Demo](#-live-demo-)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 SPACE TRAVELERS HUB <a name="Portfolio-site"></a>
+
+**Space Traveler's Hub** is an app for Space X that provides commercial and scientific space travel services. Built with React and Redux, The app allows users to reserve rockets and join selected space missions.
+
+## Built With <a name="built-with"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://redux-toolkit.js.org/">Redux</a></li>
+    <li><a href="https://rest-country-api.p.rapidapi.com/">World country API</a></li>
+  </ul>
+</details>
+
+## Tech stack <a name="tech-stack"></a>
+This project is built with: 
+ <ul>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://redux-toolkit.js.org/">Redux</a></li>
+    <li><a href="https://rest-country-api.p.rapidapi.com/">World country API</a></li>
+  </ul>
+
+<!-- Features -->
+
+###  Key Features <a name="key-features"></a>
+- Display a list current currency with brief descriptions 
+- Display a list of all available food economy with their description
+- Reserve a budget for country
+- Join or leave the details of country
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Running this project on your computer.
+
+To get a local copy up and running, follow these steps:
+- Clone this repository
+run `https://github.com/tchasinga/country-global.git`
+
+- Follow the steps in **Setup** below
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A GitHub account
+- Node JS installed on your machine for npm
+- code editor such as VSCode or Atom
+- Web browser
+
+### Setup
+
+Make a new directory in your local computer. To do this:
+
+- run `mkdir <directory-name>`
+- change the location to the directory you created 
+    run `cd <directory-name>`
+- Follow the steps in **Install** below
+
+### Install
+ To Install this project, follow the steps below:
+- Clone this repository
+run `https://github.com/tchasinga/country-global.git`
+
+- run `country-global`
+
+- run `npm start`
+
+The project will open on your default browser. Interact with the project :)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+🚀 Live Demo
+============================
+```
+Coming Soon 
+```
```

Comment:
- [ ] We can't have "coming soon" in this project as you are required to have deployed your app and provided its link on the README. Kindly do that.

- [ ] Kindly go through the project guidelines and you will find that you are expected to record a video presentation of your project, not more than 5 minutes long. Kindly record the video and provide its link on the README.